sdk/src/feature_set.rs

```diff
@@ -725,6 +725,10 @@ pub mod disable_rent_fees_collection {
     solana_sdk::declare_id!("CJzY83ggJHqPGDq8VisV3U91jDJLuEaALZooBrXtnnLU");
 }
 
+pub mod enable_transaction_failure_fees {
+    solana_sdk::declare_id!("My11111111111111111111111111111111111111111");
```

Comment:
feel free to use the grinded key I prepared: (pay me) `PaymEPK2oqwT9TXAVfadjztH2H6KfLEB9Hhd5Q5frvP`